examples/wip/viewport-animation/src/app.js

```diff
@@ -0,0 +1,101 @@
+/* global window */
+import React, {Component} from 'react';
+import {StaticMap} from 'react-map-gl';
+import {experimental} from 'deck.gl';
+import TWEEN from 'tween.js';
+
+import ControlPanel from './control-panel';
+
+const {AnimationMapController, viewportFlyToAnimation} = experimental;
+const token = process.env.MapboxAccessToken; // eslint-disable-line
+const ENABLE_ZOOM_CHANGE = false;
+const RANDOMIZE_ZOOM = false;
+
+if (!token) {
```

Comment:
Nit: This should be handled by `react-map-gl`. Test if that is the case and delete here?